pyproject.toml

```
@@ -1,3 +1,31 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=7"]
```

Would suggest breaking this over multiple lines (makes diffs that change this easier to read)

Also calling out `tomli` explicitly (and when we can drop it as the standard library includes it) may be helpful

```suggestion
requires = [
    "setuptools>=45",
    "setuptools_scm>=7",
    "tomli>=1.0.0; python_version < '3.11'",
]
```